src/basic/env-util.c

```diff
@@ -803,44 +802,3 @@ int setenv_systemd_exec_pid(bool update_only) {
 
         return 1;
 }
-
-int getenv_path_list(const char *name, char ***ret_paths) {
```

Comment:
i think this should probably stay generic and we should use it in xdg_user_dirs() and such things. There's plenty places we parse env vars that are lists of paths, and hence we should have a generic implementation i am sure